mail_debrand/tests/test_mail_debrand.py

```diff
@@ -63,5 +76,5 @@ def test_lang_paynow_debrand_partner(self):
         mail = self.env['mail.mail'].search(
             [('mail_message_id', '=', msg.id)]
         )
-        self.assertNotIn('Powered by', mail.body_html)
```

Comment:
The test was correctly looking for `'Powered by'` string because this specific test checks the behavior described in https://github.com/OCA/social/pull/736#pullrequestreview-699620680, please restore this assert